cpp/src/gandiva/engine_llvm_test.cc

```diff
@@ -100,7 +101,7 @@ llvm::Function* TestEngine::BuildVecAdd(Engine* engine, LLVMTypes* types) {
 
 TEST_F(TestEngine, TestAddUnoptimised) {
   std::unique_ptr<Engine> engine;
-  Status status = Engine::Make(ConfigurationBuilder::DefaultConfiguration(), &engine);
+  auto status = Engine::Make(TestConfiguration(), &engine);
```

Comment:
any reason for this change, from what i remember default already has the build output location from kByteCodeFilePath

Reply:
Yes. I want to use installed `irhelper.bc` instead of built `irhelper.bc`.